src/rating/rating.ts

```diff
@@ -97,9 +98,12 @@ export class NgbRating implements ControlValueAccessor,
 
   /**
    * A template to override star display.
-   * Alternatively put a <ng-template> as the only child of <ngb-rating> element
    */
-  @Input() @ContentChild(TemplateRef) starTemplate: TemplateRef<StarTemplateContext>;
+  @Input() starTemplate: TemplateRef<StarTemplateContext>;
+  /**
+   * Alternatively put a <ng-template> as the only child of <ngb-rating> element.
```

Comment:
Could you please leave the comment where it was initially? 
It's for the documentation generation here → https://ng-bootstrap.github.io/#/components/rating/api